_posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc

```diff
@@ -1593,29 +1764,132 @@ inputs = {
   vpc_id             = dependency.vpc.outputs.vpc_id
   private_subnet_ids = dependency.vpc.outputs.vpc_id
 
-  container_image = {
-    repo = dependency.ecr.outputs.url
-    tag  = "v1"
+  terraform_planner_config = {
+    container_image = {
+      docker_image = dependency.ecr.outputs.url
+      docker_tag   = "v1"
+    }
+
+    infrastructure_live_repositories = ["git@github.com:<YOUR_ORG>/refarch-demo-infrastructure-live.git"]
+    secrets_manager_env_vars         = {}
+    env_vars                         = {}
+
+    # Set this to the Secrets Manager ARN that was outputted when you created the Secrets Manager entry.
+    repo_access_ssh_key_secrets_manager_arn = "ARN_TO_SECRETS_MANAGER_ENTRY_WITH_SSH_PRIVATE_KEY"
+
+    # Include read only access for the services that you use. Here we include typical read only permissions needed for
+    # deploying EC2 clusters.
+    iam_policy = {
+      AutoScalingReadOnlyAccess = {
+        effect = "Allow"
+        actions = [
+          "autoscaling:Describe*",
+        ]
+        resources = ["*"]
+      }
+      EC2ReadOnlyAccess = {
+        effect = "Allow"
+        actions = [
+          "ec2:Describe*",
+        ]
+        resources = ["*"]
+      }
+      IAMReadOnlyAccess = {
+        effect = "Allow"
+        actions = [
+          "iam:Get*",
+          "iam:List*",
+          "iam:PassRole",
+        ]
+        resources = ["*"]
+      }
+      S3ReadOnlyAccess = {
+        effect = "Allow"
+        actions = [
+          "s3:Get*",
+          "s3:List*",
+        ]
+        resources = ["*"]
+      }
+
+      # The following permissions are needed to read the remote state resources
+      S3StateBucketAccess = {
+        effect  = "Allow"
+        actions = ["s3:*"]
+        resources = [
+          "arn:aws:s3:::${local.state_bucket}",
+          "arn:aws:s3:::${local.state_bucket}/*",
+        ]
+      }
+      DynamoDBStateBucketAccess = {
```

Comment:
```suggestion
      DynamoDBLocksTableAccess = {
```

Reply:
FYI, I've updated this in our current deployment as well.

Reply:
Ah good idea!